src/components/TransactionList/TransactionListTs.ts

```diff
@@ -209,10 +209,18 @@ export class TransactionListTs extends Vue {
    * @returns {Transaction[]}
    */
   public getCurrentTabTransactions(tabName: TabName): Transaction[] {
-    if (tabName === 'confirmed') return this.confirmedTransactions || []
-    if (tabName === 'unconfirmed') return this.unconfirmedTransactions || []
-    if (tabName === 'partial') return this.partialTransactions || []
-    return []
+    switch (tabName) {
+      case 'confirmed':
+        return this.confirmedTransactions
+      case 'unconfirmed':
+        return this.unconfirmedTransactions
+      case 'partial':
+        return this.partialTransactions
+      case 'all':
+        return [ ...this.unconfirmedTransactions,...this.partialTransactions, ...this.confirmedTransactions ]
```

Comment:
I don't see a change on **pagination**. 

What happens with : 

- 20 unconfirmed transactions
- 15 partial transactions
- 40 confirmed transactions 

Pagination must be update to work correctly in these cases. It is very likely that pagination would break if you only update the value.

Reply:
Whether to display more friendly in the drop-down list box when the style of the TabPanel is canceled
like this :
![page_1](https://user-images.githubusercontent.com/18167466/78421755-ba38b900-768c-11ea-9256-0aa45cf5b1f4.png)